root/static/scripts/edit/utility/fullwidthLatin.js

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright (C) 2018 MetaBrainz Foundation
+ *
+ * This file is part of MusicBrainz, the open internet music database,
+ * and is licensed under the GPL version 2, or (at your option) any
+ * later version: http://www.gnu.org/licenses/gpl-2.0.txt
+ */
+
+/*
+ * IDEOGRAPHIC SPACE U+3000 serves as fullwidth of ASCII space 0x20;
+ * U+FF01-FF5E are fullwidth of ASCII printable characters 0x21-7E;
+ * U+FF5F-FF60 are fullwidth white/double parenthesis U+2985-2986;
+ * U+FFE0-FFE6 are fullwidth of a few Latin-1 supplement characters.
+ */
+const fullwidthLatinRegexp = /[\u3000\uFF01-\uFF60\uFFE0-\uFFE6]/;
+
+exports.hasFullwidthLatin = function (str) {
+  return fullwidthLatinRegexp.test(str || '');
+};
+
+exports.fromFullwidthLatin = function (str) {
```

Comment:
This looks like it can also replace `fullWidthConverter` in root/static/scripts/release-editor/trackParser.js

Flow types would be nice. :)

Reply:
Added types in https://github.com/metabrainz/musicbrainz-server/pull/697/commits/6107a272cda5ec8aa84fa69abf29366923ddd330 and replaced `fullWidthConverter` in https://github.com/metabrainz/musicbrainz-server/pull/697/commits/b1d00516af0c2c9612dea291c03ea5f3ada57a8f.